src/components/Shared/FormElements/AddressFormComponents/ManualEntry.test.tsx

```diff
@@ -65,8 +65,13 @@ describe("ManualEntry", () => {
         target: { value: "test town" },
       });
 
+      fireEvent.input(screen.getByLabelText("Postcode"), {
+        target: { value: "test postcode" },
+      });
+
       expect(screen.getByDisplayValue("test address 1")).toBeInTheDocument();
       expect(screen.getByDisplayValue("test town")).toBeInTheDocument();
+      expect(screen.getByDisplayValue("test postcode")).toBeInTheDocument();
```

Comment:
now that a real postcode is required, is this causing a failure? use a real postcode format 
e.g. PA London, SW1E 5DN